web/src/ui_init.js

```diff
@@ -651,10 +651,15 @@ export function initialize_everything(state_data) {
     topic_list.initialize({
         on_topic_click(stream_id, topic) {
             const sub = sub_store.get(stream_id);
+            const latest_msg_id = stream_topic_history.get_latest_message_id_in_topic(
+                stream_id,
+                topic,
+            );
             message_view.show(
                 [
                     {operator: "channel", operand: sub.name},
                     {operator: "topic", operand: topic},
+                    {operator: "with", operand: latest_msg_id, negated: false},
```

Comment:
Do you need `negated` here? The other operators don't have it.

Reply:
no, it's actually not needed, I'll remove it

Reply:
It's not needed. However, I am not sure if this is the direction we wanted to proceed for topic clicks in left sidebar, or we want to proceed as per https://chat.zulip.org/#narrow/stream/6-frontend/topic/presence.20of.20.60with.60.20operator.20after.20processing.20the.20permalink/near/1901764

Reply:
Updated!